scripts/deploy-helm.sh

```diff
@@ -30,9 +30,42 @@ check_if_parameter_is_empty()
   fi
 }
 
-print_help()
+check_api_token()
 {
-   printf "
+  URL=$1
+  if RESPONSE=$(curl -k -s -X POST -d "{\"token\":\"$DYNATRACE_ACCESS_KEY\"}" "$URL/api/v2/apiTokens/lookup" -w "<<HTTP_CODE>>%{http_code}" -H "accept: application/json; charset=utf-8" -H "Content-Type: application/json; charset=utf-8" -H "Authorization: Api-Token $DYNATRACE_ACCESS_KEY"); then
+    CODE=$(sed -rn 's/.*<<HTTP_CODE>>(.*)$/\1/p' <<<"$RESPONSE")
+    RESPONSE=$(sed -r 's/(.*)<<HTTP_CODE>>.*$/\1/' <<<"$RESPONSE")
+    if [ "$CODE" -ge 300 ]; then
+      echo "Failed to check Dynatrace API token permissions - please verify provided URL and API token. $RESPONSE"
```

Comment:
please add value of URL in log message - it will help customer to investigate the problem